boot.py

```diff
@@ -120,7 +124,9 @@
     "LspHierarchyToggleCommand",
     "LspHoverCommand",
     "LspInlayHintClickCommand",
+    "LspInlineCompletionCommand",
     "LspNextDiagnosticCommand",
+    "LspNextInlineCompletionCommand",
```

Comment:
I am not sure how to test the `LspNextInlineCompletionCommand`

Reply:
Currently I trigger the autocomple,
that also triggers the inline completions request,
and I will see the inline completion phantom,
When I trigger the `LspNextInlineCompletionCommand`, I expect to see a different inline completion phantom, but I still see the same phantom. Do you see a different phantom when triggering the `LspNextInlineCompletionCommand`?

Reply:
In my experiments, Tabby always provides only a single inline completion item. So it's expected that currently nothing happens when you trigger that command.